shared/common-adapters/button.css

```diff
@@ -4,24 +4,39 @@
 .button__border {
   border: 1px solid rgba(0, 0, 0, 0.2);
 }
+.darkMode .button__border {
+  border: 1px solid rgba(255, 255, 255, 0.2);
+}
 .button__border_blue:hover {
   border: 1px solid rgba(77, 142, 255, 0.3);
 }
+.darkMode .button__border_blue:hover {
+  border: 1px solid rgba(178, 113, 0, 0.3);
```

Comment:
It feels a little silly to me to be specifying inverse colors here when I assume we're going to be going back and replacing them with different shades of the original color. But it's like 5 lines so /shrug

Reply:
I went back and forth on that myself, but I figured that since we'll have to put this dark mode css logic in place anyway, I might as well put it in while I was in the file.